lib/food.js

```diff
@@ -5,5 +5,3 @@ class Food{
     this.calories = foodJson.calories
   };
 }
-
-module.exports = Food
```

Comment:
I'm pretty sure exporting something is redundant when requiring it with a script tag. The browser doesn't recognize module and throws an error in the console. I simply removed it and the code works fine.

Reply:
Ive been using exportation in order for my tests to work, though this may be solved tomorrow. Our use of multiple script tags in the food.html header is probably not good form over the long run. Once again, hopefully tomorrow will help us get to the bottom of this.